crates/controller/src/hud/map.rs

```diff
@@ -0,0 +1,30 @@
+use bevy::prelude::{default, AlignSelf, Color, Component, NodeBundle, Size, Style, Val};
+
+pub(crate) struct MapBundle {
+    background: Color,
+}
+
+impl MapBundle {
+    pub(crate) fn new(color: Color) -> MapBundle {
+        MapBundle { background: color }
+    }
+
+    pub(crate) fn build(&self) -> NodeBundle {
+        NodeBundle {
+            style: Style {
+                size: Size {
+                    width: Val::Percent(100.),
+                    height: Val::Percent(100.),
+                },
+                align_self: AlignSelf::Stretch,
+                ..default()
+            },
+            background_color: self.background.into(),
+            ..default()
+        }
+    }
+}
+
+/// Node marker for a parent drawing map details.
+#[derive(Component)]
+pub(crate) struct MapNode;
```

Comment:
This does not serve any purpose at the moment. I would delete it.

Reply:
I know, I know.. YAGNI.. But this is _Marker_ Trait which should be there for adding dynamic children, based on additional _Marker_ for `object`, like `MinimapDrawableObject`, to be drawn and moved on the minimap..

Reply:
Once needed, you can add it.